src/content-library/content-library.ts

```diff
@@ -0,0 +1,34 @@
+import type Activity from '../content-types/activity';
+import Article from '../content-types/article';
+import ContentLoader from './content-loader';
+
+class ContentLibrary {
+	loader: ContentLoader;
+	articles: Map<string, Article> = new Map<string, Article>();
+	activities: Map<string, Activity> = new Map<string, Activity>();
+
+	getArticle(id: string) {
```

Comment:
These are for testing purposes (although they might also be useful in general.) I imagine this interface will grow a bit larger, e.g. `getAllActivities()`.